src/Sql/dbo/Stored Procedures/Cipher_Restore.sql

```diff
@@ -21,8 +21,9 @@ BEGIN
         [dbo].[UserCipherDetails](@UserId)
     WHERE
         [Edit] = 1
+        AND [DeletedDate] IS NOT NULL
         AND [Id] IN (SELECT * FROM @Ids)
-	
+    
```

Comment:
That's going on here? Remove whitespace?

Reply:
Yeah, I have a plug-in that swaps tabs for spaces if it detects mixed tabs/spaces. I'll try to watch erroneous changes like this or just remove that plug-in

Reply:
Seems like we have extra whitespace indention where not necessary. That's what I was pointing out. We should remove those spaces.

Reply:
Fixed